src/libs/actions/App.ts

```diff
@@ -4,6 +4,7 @@ import type {AppStateStatus} from 'react-native';
 import {AppState} from 'react-native';
 import type {OnyxCollection, OnyxEntry, OnyxUpdate} from 'react-native-onyx';
 import Onyx from 'react-native-onyx';
+import OnyxUtils from 'react-native-onyx/dist/OnyxUtils';
```

Comment:
Lint is failing here, let's remove this import

Reply:
@danieldoglas 

I checked this branch and this file has no additional eslint errors.

This line was removed in the commit: https://github.com/Expensify/App/pull/69646/commits/5272033679050b768e25ee1bcc6ae3d9ea45aa2d

Are you sure you tested the latest version of this branch?
Does it throw any other errors besides the one Błażej mentioned?

<img width="1214" height="165" alt="image" src="https://github.com/user-attachments/assets/c645ffec-f4d4-4823-8fb3-57f4dcd497e5" />

Reply:
Strange, it was showing the error on GH.

OK, please merge main here and let's merge it!